app/views/shared/_navbar.html.erb

```diff
@@ -37,12 +36,14 @@
             <% end %>
             <li class="dropdown">
               <a class="dropdown-toggle" aria-expanded="true" role="button" data-toggle="dropdown" href="#">
-                <%= t('language') %>
+                <%= image_tag("file.#{locale}.png") %>
+                <if>
                 <span class="caret"></span>
               </a>
               <ul class="dropdown-menu" role="menu">
                 <% I18n.available_locales.each do |locale| %>
-                  <li><%= link_to(locale.to_s, url_for(locale: locale), data: { no_turbolink: true }) %></li>
+                  <li id="locale_image"><%= link_to(image_tag("file.#{locale}.png") + '   ' + locale.to_s, url_for(locale: locale), data: { no_turbolink: true }) %></li>
```

Comment:
It is because we are putting not only the flag but the locale name (en or pt) along with the flag.